### DISCUSSION TITLE
Discord channel for Backstage announcements plugin

### DISCUSSION THREAD

**Discussion by kurtaking** [procore-oss/backstage-plugin-announcements#192] [Announcements]
Hey all, 

We primarily recommend using the official Backstage Discord, but we wanted to inform the community that we do have a channel in our `procore-oss` Discord server. 

You can join our Discord from our main org [page](https://github.com/procore-oss).

**Comment by drodil** [comment]
Can we get an invitation to the server? I cannot join with that link

**Reply by kurtaking** [reply]
Thanks for the heads up. Does [this](https://raw.githubusercontent.com/procore-oss/.github/main/discord_badge_scaled.svg) work? You can also find an icon to access the discord from our main GitHub [page](https://github.com/procore-oss). 

I'll update the link in my main post once you confirm

**Reply by drodil** [reply]
Link in the github page worked, thanks!

**Reply by kurtaking** [reply]
Thanks, I updated my original post.
